app/api/v01/entities/vehicle.rb

```diff
@@ -50,6 +50,8 @@ def self.entity_name
   expose(:speed_multiplicator, documentation: { type: Float, desc: 'Deprecated, use speed_multiplier instead.' }) { |m| m.speed_multiplier }
   expose(:speed_multiplier, documentation: { type: Float })
   expose(:max_distance, documentation: { type: Integer, desc: 'Maximum achievable distance in meters' })
+  expose(:max_ride_distance, documentation: { type: Integer, desc: 'Maximum riding distance between two stops within a route in meters' })
+  expose(:max_ride_duration, documentation: { type: DateTime, desc: 'Maximum riding time between two stops within a route' }) { |m| m.max_ride_duration_absolute_time_with_seconds }
```

Comment:
```suggestion
  expose(:max_ride_duration, documentation: { type: DateTime, desc: 'Maximum riding time between two stops within a route in seconds.' }) { |m| m.max_ride_duration_absolute_time_with_seconds }
```

Reply:
The method `max_ride_duration_absolute_time_with_seconds` returns a time in HH:MM:SS so the mention in seconds as no real meaning